engine/runtime-integration-tests/src/test/java/org/enso/example/PolyglotTestClass.java

```diff
@@ -0,0 +1,18 @@
+package org.enso.example;
+
+import java.time.LocalDate;
+import org.graalvm.polyglot.Value;
+
+public class PolyglotTestClass {
```

Comment:
Utility classes should have `private` constructor.

Reply:
Refactored in 78b1832830c6cdeb03026be7a08ddaceaf39fcb3